### DISCUSSION TITLE
sqlite uuid unreadable code

### DISCUSSION THREAD

**Discussion by zy97** [SeaQL/sea-orm#1369] [Q&A]
Hi，
I met a trouble,I create a table using sqlite whose primary key is uuid，as follows
`
#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
#[sea_orm(table_name = "download")]
pub struct Model {
    #[sea_orm(primary_key, auto_increment = false, column_type = "Uuid")]
    pub id: Uuid,
    pub key: String,
    pub hash: String,
    pub size: String,
    pub mime_type: String,
    pub path: String,
    pub download_date: DateTimeUtc,
}

#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {}

impl ActiveModelBehavior for ActiveModel {}
`
Inserting and reading an item works fine,but when I open the database to view, the primary key is garbled，as follows
![image](https://user-images.githubusercontent.com/20431739/210914741-e1d52096-b785-404c-8961-694044460704.png)
I expect it is a readable form

**Comment by billy1624** [comment]
Hey @zy97, UUID will be encoded into byte and stored in SQLite. This behaviour is defined by SQLx at https://docs.rs/sqlx-core/0.6.2/src/sqlx_core/sqlite/types/uuid.rs.html#20

**Reply by zy97** [reply]
Sqlx seems to be able to use Hyphenated，is it that at present, sea-orm can only use Uuid, and there is no other way to make Uuid readable in sqlite?
